habitat/datasets/vln/r2r_vln_dataset.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import gzip
+import json
+import os
+from typing import List, Optional
+
+from habitat.config import Config
+from habitat.core.dataset import Dataset
+from habitat.core.logging import logger
+from habitat.core.registry import registry
+from habitat.datasets.utils import VocabDict
+from habitat.tasks.nav.nav import NavigationGoal, ShortestPathPoint
+from habitat.tasks.vln.vln import InstructionData, VLNEpisode
+
+DEFAULT_SCENE_PATH_PREFIX = "data/scene_datasets/mp3d/"
+
+R2R_VAL_SEEN_EPISODES = 781
+
+
+@registry.register_dataset(name="R2RVLN-v1")
```

Comment:
We should register it `MP3DR2RVLN-v1` or change doc string not attach it to MP3D only. From my experience this class can be scene dataset agnostic.
```suggestion
@registry.register_dataset(name="R2RVLN-v1")
```

Reply:
I changed the doc string as the class is not tied to MP3D.